src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+/*
+ * 
+ *Parent class of address, email and phone. 
+ *Guarantees: immutable; is valid as declared in the subclasses
+ */
+
+public class Contact {
+	
+		
+	public final String value;
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.